### DISCUSSION TITLE
Token-Transformer fails to resolve with parent key

### DISCUSSION THREAD

**Discussion by Todberg** [six7/figma-tokens#1348] [Q&A]
Running the `token-transformer` on a single `tokens.json` with all token sets does not resolve aliases. Consider the following exported json with three sets: `global`, `light` and `dark`:

```json
{
  "global": {
    "spacing": {
      "scale": {
        "value": 8,
        "type": "spacing",
        "description": "Do not use this token!"
      },
      "2xs": {
        "value": "{spacing.scale} * 0.25",
        "type": "spacing"
      }
    }
  },
  "light": {...},
  "dark": {...}
}
```

As you can see, the alias is `{spacing.scale}`, however, because the json includes the parent key (`global`), it does not resolve it.

**Comment by six7** [comment]
Are you sure? the main job of token-transformer is to pick up this single .json and make these references work. What command are you running?

**Reply by Todberg** [reply]
No, the transformer works fine. Was a problem on my end.
